ts/Core/Series/SeriesDefaults.ts

```diff
@@ -2548,6 +2548,33 @@ const seriesDefaults: PlotOptionsOf<Series> = {
      * @apioption plotOptions.series.legendSymbol
      */
 
+    /**
+     * Defines the color of the legend symbol for this series.
+     *
+     * @sample {highcharts|highstock} highcharts/series/legend-color/
+     *         Change the legend symbol color
+     *
+     * @type      {Highcharts.ColorString|Highcharts.GradientColorObject|Highcharts.PatternObject}
+     * @default   undefined
+     * @since     11.4.8
```

Comment:
This version is already released ;) I think it's better to remove it.

Reply:
Tip: use `next` as the version number if you want for an option to have the version number assigned at the next version release regardless what that next version number will be.

Reply:
Thanks Kacper! Is it written down somewhere? Cause I was working on this docs more less blindly - like chat GPT imitating stuff around

Reply:
[Yes](https://github.com/highcharts/highcharts/pull/21783)